### PR TITLE
Supported thrift server(JDBC connectivity) in Astro

### PR DESCRIPTION
**Pull request by ravipesala**
Now user can use JDBC driver to connect to Astro.
```
Class.forName("org.apache.hive.jdbc.HiveDriver")
val con = DriverManager.getConnection("jdbc:hive2://localhost:10000/default", "", "");
val stmt = con.createStatement();
stmt.executeQuery("select * from testdata")
```

User can use the scripts ```bin/start-astro-thriftserver.sh ``` and ```bin/stop-astro-thriftserver.sh``` to start and stop the thrift server.

By default, the server listens on localhost:10000. You may override this bahaviour via either environment variables, i.e.:

```
export HIVE_SERVER2_THRIFT_PORT=<listening-port>
export HIVE_SERVER2_THRIFT_BIND_HOST=<listening-host>
./bin/start-astro-thriftserver.sh 
  ...
```
or system properties:
```
./bin/start-astro-thriftserver.sh \
  --hiveconf hive.server2.thrift.port=<listening-port> \
  --hiveconf hive.server2.thrift.bind.host=<listening-host> 
  ...
```

### CODE REVIEW

File: bin/start-astro-thriftserver.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+# Figure out where Spark is installed
+if [ -z "$SPARK_HOME" ]
+then
+echo "\$SPARK_HOME is not set"
+fi
+
+export FWDIR=$SPARK_HOME
+export ASTRO_SOURCE="$(cd "`dirname "$0"`"/..; pwd)"
+export JAR="$ASTRO_SOURCE/target/spark-sql-on-hbase-1.0.0.jar"
```
Comment (Zhangshunyu) [inline bin/start-astro-thriftserver.sh]: Execuse me, is the jar changed back to the original name？

Comment (ravipesala) [inline bin/start-astro-thriftserver.sh]: Version name (jar name) changed in master branch. This PR is raised on spark1.5-alpha branch.

**Comment by ravipesala** [comment]
retest this please

**Comment by ravipesala** [comment]
retest this please

**Comment by ravipesala** [comment]
retest this please

**Pull request merged**